cf_units/__init__.py

```diff
@@ -964,36 +964,6 @@ def is_time_reference(self):
         """
         return self.calendar is not None
 
-    def is_long_time_interval(self):
```

Comment:
All looks good except : for some backwards compatibility, it might be nice to deprecate this function instead of simply removing it ?

Checked : iris is currently using, as noted in the PR header notes.  
So that at least might prompt us to retain it for a grace period.

Reply:
947fbca630e6b421b443322bc68771e968f69733